classes/output/core_renderer.php

```diff
@@ -1736,6 +1736,7 @@ public function render_social_network($socialnetwork) {
                 'class' => 'socialicon ' . $socialnetwork,
                 'onclick' => "window.open('".\theme_essential\toolbox::get_setting($socialnetwork)."')",
                 'title' => get_string($socialnetwork, 'theme_essential'),
+                'aria-label' => $socialnetwork,
```

Comment:
Rejected, as using the 'key' and not the result of the get_string on that key.  Please refactor and resubmit as a single commit.